test/files/neg/t7292-removal.check

```diff
@@ -1,10 +1,10 @@
-t7292-removal.scala:2: error: Octal escape literals are unsupported, use \u0000 instead.
+t7292-removal.scala:2: error: octal escape literals are unsupported. use \u0000 instead.
   val chr1 = '\0'
               ^
-t7292-removal.scala:3: error: Octal escape literals are unsupported, use \u0053 instead.
+t7292-removal.scala:3: error: octal escape literals are unsupported. use \u0053 instead.
   val str1 = "abc\123456"
                  ^
-t7292-removal.scala:4: error: Octal escape literals are unsupported, use \n instead.
+t7292-removal.scala:4: error: octal escape literals are unsupported. use \n instead.
```

Comment:
Should error messages use normal capitalisation?

Reply:
There are several things which are correct, but neither the old version (independent clauses joined by a comma) nor the new version (two lowercase sentences following a colon) follow standard colon usage and/or grammar rules.  These are all arguably correct:

```
octal escape literals are unsupported; use \n instead.
Octal escape literals are unsupported; use \n instead.
Octal escape literals are unsupported.  Use \n instead.
```

Reply:
I didn't mind the comma splice, it's kinda hipsterish.

Reply:
But it's capitalization like civilization.

Reply:
I am used to seeing all lowercase error messages from scalac. Here are some samplings from `*.check` files:

```
abstract-class-2.scala:11: error: object creation impossible, since method f in trait S2 of type (x: P2.this.p.S1)Int is not defined

S.scala:1: error: class S needs to be abstract, since method g in class J of type (y: Int, z: java.util.List)Int is not defined

classmanifests_new_deprecations.scala:2: warning: type ClassManifest in object Predef is deprecated (since 2.10.0): use `scala.reflect.ClassTag` instead

divergent-implicit.scala:5: error: diverging implicit expansion for type Int => String

ambiguous-same.scala:13: error: reference to x is ambiguous;
it is both defined in object X and imported subsequently by
import X.x

serialversionuid-not-const.scala:1: error: annotation argument needs to be a constant; found: 13L.toLong
```

There are certainly capitalized error messages, but the classic ones are mostly all lowercase.

As per punctuations are concerned, could I interest you in colon and no-trailing-period?

```
error: octal escape literals are unsupported: use \n instead
```

This `$clause: use X instead` is the same structure as the deprecation warnings.

Reply:
I like the explanatory colon. It coordinates in both the grammar and fashion sense.